### ISSUE TITLE
Add takeoff distance and heading box for VTOL takeoff

### ISSUE THREAD

**Issue by moreba1** [mavlink/qgroundcontrol#8973]
Hi

It is good to specify a default takeoff distance value for VTOL takeoff, also if takeoff distance and heading box added to VTOL takeoff command (like landing pattern), we can set it accurately.

**Comment by Antiheavy** [comment]
I agree this seems like a good request for both VTOL and FW Takeoff Patterns.  Consistency with the Landing Pattern would be nice, plus appropriate defaults can be saved for different vehicles.
